src/Notifications/NotifierInterface.php

```diff
@@ -33,11 +33,11 @@ public function send();
     /**
      * Set params in order to construct the request.
      *
-     * @param array $params
+     * @param \stdClass $params
      *
      * @return \CachetHQ\Cachet\Notifications\NotifierInterface
      */
-    public function setParams(array $params);
+    public function setParams(\stdClass $params);
```

Comment:
I didn't want to change the behavior of `Service`. And  `Service` returns stdclass for properties because it's based on json_decode/encode